llama-index-integrations/readers/llama-index-readers-jira/llama_index/readers/jira/base.py

```diff
@@ -109,6 +109,7 @@ def load_data(self, query: str) -> List[Document]:
             issues.append(
                 Document(
                     text=f"{issue.fields.summary} \n {issue.fields.description}",
+                    doc_id=issue.id
```

Comment:
You can always customize this at your end by extending the class 🤔

There is no loss with this change too tbh

Reply:
yea I think I actually prefer when loaders set the ID like this, makes it easier to delete things later